src/nwbinspector/inspector_tools/__init__.py

```diff
@@ -9,5 +9,12 @@
 
 # Still keep imports functional with warning for soft deprecation cycle
 # TODO: remove after 9/15/2024
-from .._organization import organize_messages, _get_report_header
-from .._formatting import format_message, MessageFormatter, FormatterOptions, print_to_console, save_report
+from .._organization import organize_messages
+from .._formatting import (
+    format_messages,
+    MessageFormatter,
+    FormatterOptions,
+    print_to_console,
+    save_report,
+    _get_report_header,
+)
```

Comment:
Impressively, Mypy was even able to catch things like this (improper imports)

Quite curious why Ruff alone wasn't able to